pynumdiff/linear_model/_linear_model.py

```diff
@@ -13,16 +13,16 @@
 #########################
 # Savitzky-Golay filter #
 #########################
-def savgoldiff(x, dt, params=None, options=None, polynomial_order=None, window_size=None, smoothing_win=None):
+def savgoldiff(x, dt, params=None, options=None, poly_order=None, window_size=None, smoothing_win=None):
```

Comment:
Decided to rename this parameter, because typing out the full word can take up a bunch of space and feel unnecessary.